CorsixTH/Lua/dialogs/resizables/directory_browser.lua

```diff
@@ -70,6 +71,26 @@ function InstallDirTreeNode:createNewNode(path)
   return InstallDirTreeNode(path)
 end
 
+local function is_iso(name)
```

Comment:
Our coding style says
``` text
Function names should use CamelCase. Global function names should begin with an upper-case character, whereas locals and method names should begin with a lower-case character.
```
See also https://github.com/CorsixTH/CorsixTH/wiki/Coding-Conventions#lua-code

Reply:
Interesting; lower_snake_case is more common for our lua local functions, though both are used:
```
rg 'local function [a-z_]+_[a-z]+\(' -q --stats

155 matches
155 matched lines
94 files contained matches
285 files searched
```
vs camelCase
```
rg 'local function [a-z][a-z-A-Z]+\(' -q --stats

95 matches
95 matched lines
40 files contained matches
285 files searched
```
For completeness PascalCase:
```
rg 'local function [A-Z][a-z-A-Z]+\(' -q --stats

12 matches
12 matched lines
6 files contained matches
285 files searched
```

Reply:
I'll change them to the documented convention, and make a note to convert everything else at some point.